worker/worker.py

```diff
@@ -162,7 +165,7 @@ def worker(worker_info, password, remote):
         if "version" not in req:
             print("Incorrect username/password")
             time.sleep(5)
-            sys.exit(1)
+            worker_exit()
```

Comment:
Change while here :)
- `print("Incorrect username/password")` replaced by `sys.stderr.write("Incorrect username/password\n")`

Reply:
@tomtor Ignore my suggestion, after a worker code review we should make a dedicated PR to have more consistency in the output.